src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,33 @@
+import { Dispatch, FC, SetStateAction } from 'react';
+import cn from 'classnames';
+
+import { Errors } from '../../types/Errors';
+import { handleError } from '../../utils/handleError';
+
+interface Props {
+  error: Errors;
+  setError: Dispatch<SetStateAction<Errors>>;
```

Comment:
It will be better and more readable) Use this syntax in the future
For all such cases
```suggestion
  setError: (error: Errors) => void;
```